src/neuroglancer/layer_panel.ts

```diff
@@ -415,8 +425,24 @@ export class LayerPanel extends RefCounted {
       }
     }
   }
-
   addLayerMenu() {
     addNewLayer(this.manager, this.selectedLayer);
   }
+
+  updateItemValueDisplay() {
+    if (!this.valueElementUpdateNeeded){
+      return;
+    }
+    this.valueElementUpdateNeeded = false;
+    console.log(this.layerWidgets)
```

Comment:
You can actually just set this on the parent element --- no need to set it on each individual widget element.   Then you can use a CSS selector like:

`.neuroglancer-layer-panel.hidden .neuroglancer-layer-item-value`

or if you used: this.element.dataset.hoverValuesHidden = "true";

.neuroglancer-layer-panel[data-hover-values-hidden="true"] .neuroglancer-layer-item-value

Also, please rename to neuroglancer-hidden, or use an attribute --- because class names are global, using an unprefixed name like "hidden" could result in unintended styling if Neuroglancer is embedded in another application that uses that class name for a different purpose.

Reply:
Thanks a lot for your explanations and patience!